src/modules/catalog.js

```diff
@@ -3312,6 +3312,170 @@ class Catalog {
       return helpers.throwHttpErrorFromResponse(new Error(), response);
     });
   }
+
+  /**
+   * Retrieve metadata searchabilities
+   *
+   * @function retrieveSearchabilities
+   * @param {object} parameters - Additional parameters for retrieving metadata searchabilities
```

Comment:
Since all keys on `parameters` are optional doesn't that mean that `parameters` itself should be optional. If yes I think we need to address this on other methods as well since I noticed it in a multiple of other places

Reply:
Done here: [ae21f69](https://github.com/Constructor-io/constructorio-node/pull/155/commits/ae21f69a25c026e34e5827a7355e659ccc0f5609)